### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by lvale**
witch ->no
switch->yes

Adding info to the user on the description

### CODE REVIEW

File: libraries/AP_BoardConfig/AP_BoardConfig.cpp [hunk 1]
```diff
@@ -105,7 +105,7 @@ const AP_Param::GroupInfo AP_BoardConfig::var_info[] = {
 #if CONFIG_HAL_BOARD == HAL_BOARD_PX4
     // @Param: SAFETY_MASK
     // @DisplayName: Channels to which ignore the safety switch state
-    // @Description: A bitmask which controls what channels can move while the safety witch has not been pressed
+    // @Description: A bitmask which controls what channels can move while the safety switch has not been pressed - 0:Ch1,1:Ch2,2:Ch3,3:Ch4,4:Ch5,5:Ch6,6:Ch7,7:Ch8
```
Comment (OXINARF) [inline libraries/AP_BoardConfig/AP_BoardConfig.cpp]: I think we don't do this (or don't want to), putting values in the description, but I might be wrong

Comment (lvale) [inline libraries/AP_BoardConfig/AP_BoardConfig.cpp]: Other params have it, that's why I added. Also it becomes quite hard to setup something if you can't tell how to set it up, without having either the source code available or best case scenario the wiki.

Comment (rmackay9) [inline libraries/AP_BoardConfig/AP_BoardConfig.cpp]: I think either the @Values or @Bitmask should be used for the actual values.  MP can process these and perhaps we should be adding issues for the other ground stations that don't.

**Comment by lvale** [comment]
https://github.com/mavlink/qgroundcontrol/issues/3677

https://github.com/ArduPilot/apm_planner/issues/965

**Comment by rmackay9** [comment]
Merged without the channels listed on the end.
@lvale, by the way, I think you accidentally created an lvale-spelling branch in master.  Becareful!

**Comment by lvale** [comment]
thanks @rmackay9 I noticed that when you mentioned it, and alread asked Franciso to delete it.The web inteface created it here instead of doing so on my fork...

**Pull request closed without merging**